changelog/unreleased/move-indicators-under-resource

```diff
@@ -0,0 +1,5 @@
+Change: Move status indicators under the resource name
+
+We've moved the sharing status indicators from own column in files list under the resource name.
```

Comment:
Proposing `We've moved the sharing status indicators from an own column in the files list to a second row under the resource name.`